src/packages/@app/pages/TestflowExplorerPage/TestflowExplorerPage.ViewModel.ts

```diff
@@ -542,4 +545,27 @@ export class TestflowExplorerPageViewModel {
     this.tabRepository.createTab(adaptedRequest);
     moveNavigation("right");
   };
+
+
+
+    /**
+   * @description - updates environment tab name
+   * @param _name - new environment name
+   */
+    public updateName = async (_name: any, event = "") => {
+      const progressiveTab = createDeepCopy(this._tab.getValue());
+      if (event === "blur" && _name === "") {
+        const data = await this.testflowRepository.readTestFlow(
+          progressiveTab.id,
+        );
+        progressiveTab.name = data.name;
+      } else if (event === "") {
+        progressiveTab.name = _name;
```

Comment:
update comment environment should be replaced with testflow

Reply:
Resovled